pkg/api/types.go

```diff
@@ -754,6 +754,8 @@ type EndpointsList struct {
 type NodeSpec struct {
 	// Capacity represents the available resources of a node
 	Capacity ResourceList `json:"capacity,omitempty"`
+	// Cidr represents the IP range assigned to the node
+	Cidr string `json:"cidr,omitempty"`
```

Comment:
PodCIDR sounds best to me.  Note, I am not reviewing the PR overall - I am not 100% in touch with this facet of development.  That said, looks good.